src/client.js

```diff
@@ -1,23 +1,26 @@
-import "core-js/shim";
 import axios from "axios";
 import * as errors from "./errors";
 
+axios.defaults.adapter = "http";
+
 /**
  * A client for generating data using mockaroo.
  */
 export default class Client {
   /**
-   * Creates a new instance
-   * @param {Object} options
-   * @param {string} options.apiKey Your mockaroo api key. See http://mockaroo.com/api/docs under "Gaining Access".
-   * @param {string} [options.host='mockaroo.com'] The hostname of the mockaroo server.
-   * @param {int} [options.port=80] The port to use.
-   * @param {boolean} [options.secure=true] Set to false to use http instead of https
-   */
-  constructor(options) {
-    var defaults = {
-      host: "api.mockaroo.com",
+     * Creates a new instance
+     * @param {Object} options
+     * @param {string} options.apiKey Your mockaroo api key. See http://mockaroo.com/api/docs under "Gaining Access".
+     * @param {string} [options.host='mockaroo.com'] The hostname of the mockaroo server.
+     * @param {int} [options.port=80] The port to use.
+     * @param {boolean} [options.secure=true] Set to false to use http instead of https
+     */
+  constructor (options) {
+    const defaults = {
+      host: "mockaroo.com",
```

Comment:
```suggestion
      host: "api.mockaroo.com",
```
This should be api.mockaroo.com, not the apex domain.

Reply:
Nice catch! Thank you!